ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/ImageOverviewTable.tsx

```diff
@@ -383,6 +401,15 @@ function ImageOverviewTable({
                                         />
                                     </Td>
                                 </Tr>
+                                {labels.length !== 0 && (
+                                    <Tr>
+                                        <Td colSpan={colSpan} style={{ paddingTop: 0 }}>
+                                            <LabelGroup isCompact numLabels={labels.length}>
+                                                {labels}
+                                            </LabelGroup>
+                                        </Td>
+                                    </Tr>
+                                )}
```

Comment:
Not a request for change, but does this muddy the semantics of rows in the table? Maybe this is only relevant to a screen reader perspective, but it seems like having `<tr>` elements containing tabular data with the occasional `<tr>` spliced in containing labels may be confusing.

Visually, I think this makes sense though. Something for the mental slow cooker or maybe relevant to discuss in office hours?

I think there were places too where e2e tests assumed even pairs of `<tr>` elements, one for data follow by one for an expanded row. Not an issue here since all of the e2e tests are green, but something we should watch out for in the future.